mmv1/templates/terraform/examples/firebase_app_hosting_backend_basic.tf.tmpl

```diff
@@ -0,0 +1,76 @@
+# Service account setup only required once per project.
+resource "google_service_account" "service_account" {
+  provider = google-beta
+
+  project = "{{index $.TestEnvVars "project_id"}}"
+
+  # Must be firebase-app-hosting-compute
+  account_id                   = "firebase-app-hosting-compute"
```

Comment:
This should use `vars` so that it gets sweepable prefix/suffix added

Reply:
This actually must be `firebase-app-hosting-compute` (see comment in the previous line). Otherwise, the experience in other surfaces (such as the Firebase Console UI) will be broken.  This is publicly documented at https://firebase.google.com/support/guides/service-accounts

Reply:
Ah sorry, thanks for clarifying!